### PR TITLE
Sbar 34 dockerize

### PR DESCRIPTION
**Pull request by holingpoon**
Hello, this PR includes @nodanaonlyzuul's PR to skip :development gems installations. Please review!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 config/*.json
+config/.env*
```
Comment (nodanaonlyzuul) [inline .gitignore:2]: Would you be able to:

1.  Commit a `./config/.env.example` with the env vars we'll want?
1.  Add an example `docker build --no-cache` command to the README?

File: run-reports.sh [hunk 2]
```diff
@@ -1,2 +1,2 @@
-ruby /app/script/create_analytics_report.rb -i ga:84655146 -a /app/config/google_auth.json -s 2017-11-01 -e 2017-11-02 -o /app
-cat /app/output_2017-11-01_2017-11-02.csv
```
Comment (nodanaonlyzuul) [inline run-reports.sh:2]: 1.  Do we need to cat it?
1.  I think  it's a safer bet to run `/ruby/app/script...` right? Maybe I'm missing something.

**Review by nodanaonlyzuul — CHANGES_REQUESTED** [review]
Awesome work @holingpoon.
I added a few notes/questions that I think will be good - let me know if you have questions about issues with them.

File: run-reports.sh (hunk 2 above)
Comment (holingpoon) [reply run-reports.sh:2]: You're right. Going to remove `cat` statement.

File: .gitignore (hunk 1 above)
Comment (holingpoon) [reply .gitignore:2]: 1. Will do.
2. Will do. 😄

**Review by nodanaonlyzuul — APPROVED** [review]

**Pull request merged**